chapters/synchronization.txt

```diff
@@ -789,12 +789,12 @@ endif::VK_KHR_depth_stencil_resolve[]
     operation.
     Accesses of this type are not performed through a resource, but directly
     on memory.
-  * ename:VK_ACCESS_MEMORY_READ_BIT is equivalent to the logical OR of
-    every other etype:READ access flag bit that is supported in the
-    pipeline stage it is used with.
-  * ename:VK_ACCESS_MEMORY_WRITE_BIT is equivalent to the logical OR of
-    every other etype:WRITE access flag bit that is supported in the
-    pipeline stage it is used with.
+  * ename:VK_ACCESS_MEMORY_READ_BIT specifies all read accesses.
+    It is always valid in any access mask, and is treated as equivalent to setting
```

Comment:
On the other hand this may be too brittle.
If some VU said the access mask must be `0`, then it would not be "always" valid.

Reply:
> On the other hand this may be too brittle.
> If some VU said the access mask must be 0, then it would not be "always" valid.

Yea I had this thought too, to the point that I figured out the one time this happens in the spec currently - if for some reason you were to submit a command buffer to a sparse-binding-only queue. It being valid remains in-line with the existing description of it, so it's fine like this IMO.

> PS: should be "any access mask" -> "any access mask parameter"?

We refer to "access mask" as a thing many other places in this chapter, so I think this is also fine.

Reply:
As per comment below, WG agreed with my assertions.